assets/src/edit-story/app/media/media3p/api/apiFetcher.js

```diff
@@ -71,6 +72,13 @@ function validatePageSize(pageSize) {
   throw new Error('Invalid page_size: ' + pageSize);
 }
 
+function validateRegisterUsagePayload(payload) {
+  if (payload && !/^\s*$/.test(payload)) {
```

Comment:
Is it necessary to validate the payload, considering we always receive one and the server would respond with an http 400 response anyway?

Reply:
It might not bring a lot of value but it at least somewhat ensures that the method is called properly. I now also instead validate that the URL contains the proper path.